sources/packages/api/src/services/education-program/education-program.service.models.ts

```diff
@@ -44,6 +44,7 @@ export interface SaveEducationProgram {
   intlExchangeProgramEligibility?: string;
   programDeclaration: boolean;
   programStatus: ProgramStatus;
+  fieldOfStudyCode: number;
```

Comment:
nullable? `fieldOfStudyCode?: number;`

Reply:
Updated.

Reply:
Why is it nullable here when it is not nullable at entity?

Reply:
It is not supposed to be nullable.